main.js

```diff
@@ -70,6 +70,14 @@ function populateDom(list) {
   addEvents();
 }
 
+function nav(element, link) {
+  document.querySelector('.show').className = 'hidden';
+  document.querySelector(element).className = 'show';
+  document.querySelectorAll('a').forEach((item) => {
+    item.className = '';
+  });
+  link.className = 'active';
+}
 // get input from form and call addToBooks method
```

Comment:
- [ ] [OPTIONAL] - Please, consider to delete all comments from to make your code more clear 💯